tests/test_indexing.py

```diff
@@ -48,7 +51,8 @@ def test_window():
         assert src.index(left+400, top-400) == (1, 1)
         assert src.index(left+dx+eps, top-dy-eps) == (1, 1)
         assert src.window(left, top-400, left+400, top) == ((0, 2), (0, 2))
-        assert src.window(left, top-2*dy-eps, left+2*dx-eps, top) == ((0, 2), (0, 2))
+        assert src.window(left, top-2*dy-eps, left+2*dx-eps,
```

Comment:
This seems like a case where pep8 is letting us down; I think we shouldn't let line wrapping rules outweigh readability of our logic.  How about:

```
window = src.window(left, top-2*dy-eps, left+2*dx-eps, top)
assert window == ((0, 2), (0, 2))
```